### PR TITLE
Changed name of hop addition

### PR DESCRIPTION
**Pull request by JamFfm**
to make it more clear,  Changed default temp to 99 therefore the timer starts earlier, add info.

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -12,24 +12,24 @@ class BoilStepWithCountdownReminders(StepBase):
...
+        "Prepare fermenter              as minutes before end"        
     ]
     # Properties
-    temp = Property.Number("Temperature", configurable=True, default_value=100, description="Target temperature for boiling")
+    temp = Property.Number("Temperature", configurable=True, default_value=99, description="Target temperature for boiling")
     kettle = StepProperty.Kettle("Kettle", description="Kettle in which the boiling step takes place")
     timer = Property.Number("Timer in Minutes", configurable=True, default_value=90, description="Timer is started when target temperature is reached")
 
-    reminder_00 = Property.Number(REMINDER_NAMES[0], configurable=True)
+    reminder_00 = Property.Number(REMINDER_NAMES[0], configurable=True, description="Fill in times like in MMM recepies, e.g. Boil for 80 Minutes fill in 80")
```
Comment (cgspeck) [inline __init__.py]: What is a "MMM recipe"?

File: __init__.py [hunk 2]
```diff
@@ -12,24 +12,24 @@ class BoilStepWithCountdownReminders(StepBase):
...
-        "Hop 3 Addition",
-        "Hop 4 Addition",
-        "Hop 5 Addition",
-        "Prepare yeast",
-        "Prepare finings",
-        "Prepare cooling system",
-        "Dose finings",
-        "Start hot-side cooling loop",
-        "Prepare fermenter"
+        "Hop 1 Addition                 as minutes boiltime to the end",
```
Comment (cgspeck) [inline __init__.py]: I tested this and the admin looks good on desktop and mobile, but it doesn't quite make sense when the alerts themselves pop up, e.g.:

image

I would suggest splitting the text `" as minutes boiltime to the end"` from the `REMINDER_NAMES`, and adding it back in where the properties are declared, e.g.:

1) declare a variable `NAME_EPILOGUE = as minutes boiltime to the end`, 
2) make sure `reminder_00` uses it, e.g. :

```
reminder_00 = Property.Number(REMINDER_NAMES[0] + NAME_EPILOGUE, configurable=True ...
```
3) and update each of the other reminders accordingly.

**Review by cgspeck — CHANGES_REQUESTED** [review]
Thank you but there is an issue with the text of the reminders when they pop up which ought to be an easy fix, please see the comments in the code. 
:beers: :smile:

File: __init__.py (hunk 2 above)
Comment (JamFfm) [reply __init__.py]: You are right. Did not take into account that the GUI is used in a mobile too. 
Maybe just using the info should be clear enough. 
 Your suggestion is fine too. 
Thanks for building this add on.

File: __init__.py (hunk 1 above)
Comment (JamFfm) [reply __init__.py]: MMM = Maische Malz und Mehr https://www.maischemalzundmehr.de/index.php

**Comment by JamFfm** [comment]
Please have a look at the changes

**Comment by cgspeck** [comment]
Thank you. I will test this on desktop & mobile, this weekend at the latest.

**Review by cgspeck — APPROVED** [review]

**Comment by cgspeck** [comment]
That's great thank you. :+1: :beers:

**Comment by JamFfm** [comment]
Thanks and keep on building nice addons. 


Von meinem iPhone gesendet

Grüße Jan Battermann

**Pull request merged**